upup/pkg/fi/cloudup/awstasks/autoscalinggroup.go

```diff
@@ -363,8 +366,8 @@ func (v *AutoscalingGroup) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *Autos
 				},
 				LaunchTemplate: &autoscaling.LaunchTemplate{
 					LaunchTemplateSpecification: &autoscaling.LaunchTemplateSpecification{
-						LaunchTemplateName: e.LaunchTemplate.ID,
-						Version:            aws.String("1"),
+						LaunchTemplateName: e.LaunchTemplate.Name,
```

Comment:
Perhaps specify the `LaunchTemplateID` instead?

Reply:
Updated, should make things clearer in case of mismatches.